proto/lbm/token/v1/tx.proto

```diff
@@ -274,7 +274,7 @@ message MsgModify {
   // the address of the grantee which must have modify permission.
   string owner = 2;
   // changes to apply.
-  // possible attribute keys are: name, img_uri, meta
+  // possible attribute keys are: name, uri, img_uri (deprecated), meta
```

Comment:
Can the `img_url` key be used after this PR merge?

Reply:
In the current implementation, yes.